src/menu/stories.js

```diff
@@ -27,26 +44,108 @@ const ITEMS: Array<{label: string}> = [
   {label: 'Item Twelve'},
 ];
 
+const profileItem = {
+  title: 'David Smith',
+  subtitle: 'Senior Engineering Manager',
+  body: 'Uber Everything',
+  imgUrl: 'https://via.placeholder.com/60x60',
+};
+const PROFILE_ITEMS = [
+  {...profileItem},
+  {...profileItem, subtitle: null},
+  {...profileItem, body: null},
+  {...profileItem, title: null},
+  {...profileItem, imgUrl: null},
+  {...profileItem, imgUrl: CloudComponent},
+];
+
 storiesOf('Menu', module)
-  .add('Stateless', () => (
+  .add('Stateless simple', () => (
     <Menu
       items={ITEMS}
-      getItemLabel={item => item.label}
       rootRef={React.createRef()}
       overrides={{
-        // $FlowFixMe
-        List: ListMaxWidth,
+        List: {
+          style: {
+            width: '200px',
+          },
+        },
+        Option: {
+          props: {
+            getItemLabel: item => item.label,
+          },
+        },
+      }}
+    />
+  ))
+  .add('Stateless profile', () => (
+    <Menu
+      items={PROFILE_ITEMS}
+      rootRef={React.createRef()}
+      overrides={{
+        List: {
+          style: {
+            width: '350px',
+          },
+        },
+        Option: {
+          component: OptionProfile,
+          props: {
+            getProfileItemLabels: ({title, subtitle, body}) => ({
+              title,
+              subtitle,
+              body,
+            }),
```

Comment:
This sort of just looks like an identity function – is it just for demonstration purposes or is this actually required?

Reply:
It's required, in this case the input `item` has `title, subtitle, body` as object keys, but in other scenarios it could be

```js
getProfileItemLabels: ({name, jobTitle, selfDescription}) => ({title: name, subtitle: jobTitle, body: selfDescription})
```